src/content/api/logging.md

```diff
@@ -21,19 +22,19 @@ Benefits of custom logging API in webpack:
 - CLI, UI tools for webpack may choose different ways to display logging
 - webpack core can emit logging output, e.g. timing data
 
-By introducing webpack logging API we hope to unify the way webpack plugins and loaders emit logging messages and allow better ways to inspect build problems. Integrated logging solution supports plugins and loaders developers by improving their development experience. Paves the way for non-CLI webpack solutions like dashboards or other UIs.
+By introducing webpack logging API we hope to unify the way webpack plugins and loaders emit logs and allow better ways to inspect build problems. Integrated logging solution supports plugins and loaders developers by improving their development experience. Paves the way for non-CLI webpack solutions like dashboards or other UIs.
 
-W> __Avoid noise in the log!__ Keep in mind that multiple plugins and loaders are used together. Loaders are usually processing multiple files and are invoked for every file. Choose logging level as low as possible to keep the log output informative.
+W> __Avoid noise in the log!__ Keep in mind that multiple plugins and loaders are used together. Loaders are usually processing multiple files and are invoked for every file. Choose the logging level as low as possible to keep the log output informative.
```

Comment:
Isnt it an 'a' ?

Reply:
I think 'the' is fine

Reply:
after reading some examples i still think the right way to say it is: `choose a level`, need someone to confirm either

Reply:
Yeah, `Choose a logging level` would be better :+1: